VENDORS.md

```diff
@@ -24,67 +25,64 @@ If you notice any mistakes, or would like to contribute, feel free to **make** a
 | [MechCables](https://mechcables.com/)                        |          |      |       | [x]    |         |           | Virginia                   |
-| [Mechwild](https://mechwild.com/)<sup>1</sup>                | [x]      |      |       |        |         | [x]       | Ohio                       |
+| [Mechwild](https://mechwild.com/)<sup>1</sup>                | [x]      |      | [x]   |        |         | [x]       | Ohio                       |
 | [Mekibo (TX Keyboards)](https://mekibo.com/)                 | [x]      | [x]  | [x]   |        | [x]     | [x]       | Garden Grove, CA           |
 | [Mimic Cables](https://mimic-cables.com/)                    |          |      |       | [x]    |         |           | ?                          |
 | [MKUltra](https://mkultra.click/)                            | [x]      |      |       | [x]    | [x]     | [x]       | Angels Camp, CA            |
 | [NovelKeys_](https://novelkeys.xyz/)                         | [x]      | [x]  | [x]   |        | [x]     | [x]       | Morgantown, WV             |
-| [OriginativeCo](https://originativeco.com/)                  |          |      |       |        | [x]     |           | Irvine, CA                 |
+| [OriginativeCo](https://originativeco.com/)                  | [x]      |      | [x]   |        | [x]     | [x]       | Irvine, CA                 |
 | [Paramountkeeb](https://paramountkeeb.com/)                  | [x]      | [x]  | [x]   |        |         | [x]       | ChickenMan's Basement, CA  |
 | [Prevail KeyCo](https://prevailkeyco.com/)                   | [x]      | [x]  | [x]   |        |         |           | Florida                    |
 | [PrimeKB](https://primekb.com/)                              | [x]      | [x]  | [x]   |        | [x]     | [x]       | Aledo, TX                  |
-| [RGBKB](https://rgbkb.net/)                                  | [x]      | [x]  | [x]   | [x]    | [x]     | [x]       | Connecticut                |
-| [Ringer Keys](https://ringerkeys.com/)                       | [x]      | [x]  | [x]   |        | [x]     | [x]       | Greater Boston, MA         |
-| [Rockets Cables](https://rocketscables.com/)                 |          |      |       | [x]    |         |           | ?                          |
-| [Space Cables](https://spaceholdings.net/)                   |          |      |       | [x]    |         |           | Austin, TX                 |
+| [RGBKB](https://rgbkb.net/)                                  | [x]      |      |       |        | [x]     | [x]       | Connecticut                |
+| [Ringer Keys](https://ringerkeys.com/)                       | [x]      | [x]  | [x]   |        |         | [x]       | Greater Boston, MA         |
+| [Space Cables](https://spaceholdings.net/)                   |          | [x]  | [x]   | [x]    | [x]     |           | Austin, TX                 |
```

Comment:
does spaceholdings have any keysets in stock, or is it all groupbuys/preorders?

Reply:
I debated this a lot internally but I decided yes because of Glacier (those UV printed clear caps from angrymiao). They are technically extras but niche enough that I think stock will last a while(?). But I think there's a strong case to not count that too